graphql/schema/types/project.graphql

```diff
@@ -156,48 +156,48 @@ Project models single repository on GitHub.
 """
 type Project {
   id: String!
-  admins: [String]
-  batchTime: Int!
+  admins: [String] @restrictProjectAccess
+  batchTime: Int! @restrictProjectAccess
   branch: String!
-  buildBaronSettings: BuildBaronSettings!
-  commitQueue: CommitQueueParams!
-  deactivatePrevious: Boolean
-  disabledStatsCache: Boolean
-  dispatchingDisabled: Boolean
+  buildBaronSettings: BuildBaronSettings! @restrictProjectAccess
+  commitQueue: CommitQueueParams! @restrictProjectAccess
```

Comment:
Don't know a lot about where directives are used but will this affect someone's ability to load the commit queue page or anything like that?

Reply:
To get the commit queue, we use the query in the screenshot below which doesn't request any fields from a `ProjectRef` object so it should be okay (I double checked this manually). Any user should be able to see any project's commit queue.
 
<img width="400" alt="Screen Shot 2023-02-07 at 10 26 48 AM" src="https://user-images.githubusercontent.com/47064971/217288000-d1fb27eb-b325-4c16-9956-42c5d08909f1.png">

We only use basic `ProjectRef` fields (e.g. `repo`, `owner`) on other pages which should not be blocked by the directive.